src/sidebar/store/test/util-test.js

```diff
@@ -105,23 +130,39 @@ describe('reducer utils', function() {
 
       assert.deepEqual(newState, { firstCounter: 6, secondCounter: 11 });
     });
+
+    it('creates a reducer that can consume a state that returns an array', function() {
```

Comment:
```suggestion
    it('supports reducer functions that return an array', function() {
```

I had trouble parsing this as "a state" is just an object, it doesn't return anything.

Reply:
better :)